diff --git a/node_modules/@types/uni-app/lib/uni.d.ts b/node_modules/@types/uni-app/lib/uni.d.ts
index aa1496c..6dbbd9c 100755
--- a/node_modules/@types/uni-app/lib/uni.d.ts
+++ b/node_modules/@types/uni-app/lib/uni.d.ts
@@ -4,6 +4,18 @@
 declare const uni: Uni;
 
 declare class Uni {
+     /**
+   * 向客户端发送事件，H5中会自动通过条件编译排除
+   * @param eventName 事件名称
+   * @param params 发送参数
+   * @param callback 回调
+  * @returns
+   */
+     sendNativeEvent: (eventName: string, params: Record<string, any>, callback: (result: any) => void) => void;
+     /**
+      * 小程序监听事件
+      */
+     onNativeEventReceive:(callback:(event:string,data:any)=>void)=>void;
     /**
      * 将 Base64 字符串转成 ArrayBuffer 对象
      *
@@ -21,7 +33,7 @@ declare class Uni {
      *
      * 参考: [http://uniapp.dcloud.io/api/window/communication?id=on](http://uniapp.dcloud.io/api/window/communication?id=on)
      */
-    $on(eventName?: string, callback?: () => void): void;
+    $on(eventName?: string, callback?: (params:any) => void): void;
     /**
      * 触发自定义事件，附加的参数会传递给事件监听器。
      *
@@ -33,7 +45,7 @@ declare class Uni {
      *
      * 参考: [http://uniapp.dcloud.io/api/window/communication?id=once](http://uniapp.dcloud.io/api/window/communication?id=once)
      */
-    $once(eventName?: string, callback?: () => void): void;
+    $once(eventName?: string, callback?: (params:any) => void): void;
     /**
      * 移除自定义事件监听器。如果没有指定事件名，则移除所有事件监听器。如果提供事件名，则移除该事件的所有监听器。如果提供了事件名和回调，则只移除这个回调的监听器。
      *
